src/components/Task/Modal/Form/Inputs/SuggestedContactStatus/SuggestedContactStatus.tsx

```diff
@@ -58,14 +59,13 @@ export const SuggestedContactStatus: React.FC<SuggestedContactStatusProps> = ({
     if (currentContactStatus) {
       const disabledStatuses: StatusEnum[] = getContactStatusesByPhase(
         PhaseEnum.PartnerCare,
-        contactStatuses,
-      ).map((s) => s.id);
+      );
       // Hide suggestedStatus if the suggested status is a partner care status, otherwise, show it
       return !disabledStatuses.includes(currentContactStatus);
     }
     // Show suggestedStatus if the contact has no status
     return true;
-  }, [contactStatuses, currentContactStatus]);
+  }, [currentContactStatus]);
```

Comment:
Can we add `suggestedContactStatus` to the dependency array? I think we also need to do `useCallback` on `getContactStatusesByPhase` and add `getContactStatusesByPhase` to the dependency array. Otherwise, `getContactStatusesByPhase` could be using the old constants data.